forc-plugins/forc-doc/src/render/sidebar.rs

```diff
@@ -59,9 +62,12 @@ impl Renderable for Sidebar {
                 format!("{} {}", title.item_title_str(), name.as_str())
             }
         };
-        let root_path = self
-            .module_info
-            .to_html_shorthand_path_string(INDEX_FILENAME);
+        let root_path = self.module_info.to_html_shorthand_path_string(
+            PathBuf::from(self.module_info.project_name())
+                .join(INDEX_FILENAME)
+                .to_str()
+                .unwrap(),
+        );
```

Comment:
Nit: is it possible to use `ok_or_else` + `?` rather than `unwrap()`? If somehow `project_name()` contains non-valid unicode (not sure if that is checked anywhere, if so feel free to disregard this comment), it would be great to return an error rather than `panic`.

If the check is done while building, we can use `.expect("Invalid project name: containing ")` to panic with some context, if you would like to